packages/domain/lib/model/user/user_model.dart

```diff
@@ -0,0 +1,30 @@
+class UserModel {
+  final String email;
+  final String password;
+
+  UserModel(
+    this.email,
+    this.password,
+  );
+
+  factory UserModel.fromJson(
+      Map<String, dynamic> json,
+      ) =>
+      UserModel(
+        json['email'],
+        json['password'],
+      );
+
+  factory UserModel.fromFacebookJson(
+      Map<String, dynamic> json,
+      ) =>
```

Comment:
```suggestion
  ) =>
```

seems that after formatting should be like this, no?